### DISCUSSION TITLE
Upgrading from 2.0.0 to latest release (2.20.1) or to 2.11 (stable Transactions API)

### DISCUSSION THREAD

**Discussion by lefam** [prisma/prisma#6481] [Q&A] [answered]
Hi.
We are using **Prisma 2.0.0** in **production** for more than a year now. We are looking to use the new **Transactions API** introduced in more recent versions.

Looking into the release history, I see there are some (small?) breaking changes starting from 2.12. There, I also see that the Transactions API got stable on **version 2.11**, so to be on safe side I am thinking on **upgrading to 2.11** and thus have access to the stable Transactions API and probably less headaches from any breaking issues from more recent versions.

What are your thoughts around this? Wondering what was the experience from others from upgrading to somewhat distant versions as in our case. Would you suggest upgrading from 2.0.0 straight to 2.20.1? Or going to 2.11 that has stable Transactions API support?

Thanks in advance.

**Comment by ryands17** [comment, marked as answer]
@lefam 👋 

In this case, it would be best to update to the latest version i.e. `2.20.1`. We have introduced a [Codemod](https://www.prisma.io/docs/concepts/more/codemod) for the same that will update all your files with the correct and latest syntax.

I would also suggest updating `typescript` to 4.1+ as the latest version of Prisma requires that.

**Reply by lefam** [reply]
Thanks.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
